doc/conf.py

```diff
@@ -76,5 +76,227 @@
 
 # -- Options for man output -------------------------------------------------
 
+# Run the gen-manlines.sh script to generate the following text
```

Comment:
Is `gen-manlines.sh` something you wrote or an external project?  If the former, should it be included with this commit?  If the latter, can a link to it be added to this comment?

Reply:
I was originally going to include the script, but then I decided not to... do you think a script to generate these lines would be useful? Or, since man pages are added a few at a time, just have people write the lines by hand?

Reply:
Anything that can automate a manual process is good IME, since every manual process will generate an error at some point.

Also if the file is fully generated by a script, the script could be included in version control but not the result.

Reply:
Since these lines are only generated once, I’m not inclined to put the generating script in. Really, the script is more a part of the conversation from adoc, rather than something that will be used repeatedly.

Reply:
Ah, ok, if the file never needs to be touched again, no reason to include the script!